src/CommunityToolkit.Maui.Camera/CameraManager.macios.cs

```diff
@@ -241,11 +248,56 @@ protected virtual void Dispose(bool disposing)
 			captureInput?.Dispose();
 			captureInput = null;
 
+			orientationDidChangeObserver?.Dispose();
+			orientationDidChangeObserver = null;
+
 			photoOutput?.Dispose();
 			photoOutput = null;
 		}
 	}
 
+	// TODO: Move GetVideoOrientation() to Extensions class?
```

Comment:
I think it's fine here if we don't need it anywhere else. Can we remove the comment if you don't want to move it?

Reply:
Done